src/couch/test/couch_db_mpr_tests.erl

```diff
@@ -69,34 +70,36 @@ couch_db_mpr_test_() ->
                 foreach,
                 fun setup/0,
                 fun teardown/1,
-                [{with, [
+                [
```

Comment:
Hm, by removing this with, don't you lose the Url that is passed to `recreate_with_mpr()`? It's late and I may not be reading this right.